uaa/src/test/java/org/cloudfoundry/identity/uaa/degraded/DegradedSamlLoginTests.java

```diff
@@ -297,8 +317,11 @@ private String getRequestUrlFromHarLogEntry(LogEntry logEntry)
             throws IOException {
 
         Map<String, Object> message = JsonUtils.readValue(logEntry.getMessage(), new TypeReference<Map<String,Object>>() {});
+        System.out.println("message:"+ message);
         Map<String, Object> log = (Map<String, Object>) message.get("log");
```

Comment:
Remove all System.out statements. Make log statements if you think they are necessary.